manifests/init.pp

```diff
@@ -33,10 +33,9 @@
   include '::nodejs::install'
 
   if $manage_package_repo {
-    include $repo_class
-    anchor { '::nodejs::begin': }
-    -> Class[$repo_class]
+    contain $repo_class
```

Comment:
Including it is better. Say a user has ```Class['::epel'] -> Class['::x'] -> Class['::nodejs']``` then trying to ```contain ::epel`` here will lead to issues while it shouldn't be. If the installation is contained and you set the repo class as a dependency then you're set. We don't care where epel is (as long as it's before us) and we don't give a guarantee about it.

Reply:
What was the anchor class around it for before then? Is it not needed, even if we switch back to `include`?

Reply:
See if the version I just pushed looks better

Reply:
I think the previous implementation was incorrect and this is the correct one.